openapi3.yaml

```diff
@@ -722,6 +749,16 @@ components:
         $ref: '#/components/schemas/task'
     task:
       type: object
+      required:
```

Comment:
if you force all fields as required rename task to taskResponse and tasks to tasksResponse

Reply:
done